doc/api/fs.md

```diff
@@ -2149,7 +2145,7 @@ fs.mkdtemp(tmpDir, (err, folder) => {
   if (err) throw err;
   console.log(folder);
   // Will print something similar to `/tmpabc123`.
-  // Note that a new temporary directory is created
+  // A new temporary directory is created
```

Comment:
Nit: can fit on three lines:
```js
  // Will print something similar to `/tmpabc123`.
  // A new temporary directory is created at the file system root
  // rather than *within* the /tmp directory.
```

Reply:
Wrapped and reduced to 3 lines. Thanks.